openlibrary/plugins/openlibrary/js/carousels.js

```diff
@@ -82,40 +80,77 @@ const Carousel = {
                 '"aria-hidden="false" role="option">' +
                 '<div class="book-cover">' +
                   '<a href="' + work.key + '" ' + isClickable + '>' +
-                    '<img class="bookcover" width="130" height="200" title="' + work.title + '" ' +
-                         'src="//covers.openlibrary.org/b/id/' + work.cover_id + '-M.jpg">' +
+                    '<img class="bookcover" width="130" height="200" title="' +
+                      work.title + '" ' +
+                      'src="//covers.openlibrary.org/b/id/' + cover_id + '-M.jpg">' +
                   '</a>' +
                 '</div>' +
                 '<div class="book-cta">' +
-                  '<a class="btn cta-btn ' + cls + '" href="' + url + '" data-ol-link-track="subjects" ' +
```

Comment:
Please use HTML template strings here to make this more readable.
There is also a big danger of HTML injection here (e.g. work.title) since we don't properly escape.